content/app/how-tos/ingress-with-whitelist.md

```diff
@@ -0,0 +1,76 @@
++++
+title = "Create Ingress with IP Whitelisting"
+weight = 5
+chapter = false
+pre = ""
++++
+
+{{% notice warning %}}
```

Comment:
assuming all this formatting is correct, who parses it? as cannot parse that on my intelijs or git properly